Cesium.CodeGen/Ir/Expressions/FunctionCallExpression.cs

```diff
@@ -36,6 +36,28 @@ public FunctionCallExpression(Ast.FunctionCallExpression expression)
 
     public IExpression Lower(IDeclarationScope scope)
     {
+        if (_function.Identifier == "__builtin_offsetof_instance")
```

Comment:
Another note for self: let's open an issue on extracting the builtin list to some special place.